doc/neps/nep-0048-spending-project-funds.rst

```diff
@@ -0,0 +1,421 @@
+.. _NEP48:
+
+=====================================
+NEP 48 — Spending NumPy Project Funds
+=====================================
+
+:Author: Ralf Gommers <ralf.gommers@gmail.com>
+:Author: Inessa Pawson <inessa@albuscode.org>
+:Author: Stefan van der Walt <stefanv@berkeley.edu>
+:Status: Draft
+:Type: Informational
+:Created: 2021-02-07
+:Resolution:
+
+
+Abstract
+--------
+
+The NumPy project has historically never received significant **unrestricted**
+funding. However, that is starting to change.  This NEP aims to provide
+guidance about spending NumPy project unrestricted funds, by formulating a set
+of principles about *what* to pay for and *who* to pay. It will also touch on
+how decisions regarding spending funds get made, how funds get administered,
+and transparency around these topics.
+
+
+Motivation and Scope
+--------------------
+
```

Comment:
```suggestion
NumPy is a fiscally sponsored project of NumFOCUS, a 501(c)(3) nonprofit organization headquartered in Austin, TX. Therefore, for all legal and accounting matters the NumPy project has to follow the rules and regulations for US nonprofits. All nonprofit donations are classified into two categories: **unrestricted funds** which may be used for any legal purpose appropriate to the organization and **restricted funds**, monies set aside for a particular purpose (e.g., project, educational program, etc.). It’s important to note that restricted funds are **permanently restricted** to that purpose and cannot be used for any other expenses.  

For the detailed timeline of NumPy funding refer to [NumPy funding – history and current status](#history-and-current-status).
```

Reply:
This reads much much better now for the rest of the history to follow.

Reply:
Thanks @InessaPawson. I'm not sure the "permanently restricted" bit is technically true or necessary, probably best left out. But for now this added paragraph helps people understand the rest better, so I'll commit it as-is.

Reply:
@rgommers You are right, it's better to leave the fourth sentence out.